README.md

```diff
@@ -173,15 +132,7 @@ Updates `betterdiscordctl` to the latest version available on GitHub.
 
 * `$XDG_DATA_HOME/betterdiscordctl` (fallback `~/.local/share/betterdiscordctl`)
 
-  `betterdiscordctl`'s machine-specific data directory.
-
-* `$XDG_DATA_HOME/betterdiscordctl/bd_map`
-
-  A mapping of current Discord installations to BetterDiscord clones.
-
-* `$XDG_DATA_HOME/betterdiscordctl/bd`
-
-  A directory of BetterDiscord clones, indexed by `bd_map`.
+  **No longer used and may be deleted in a later release.**
```

Comment:
Leave this to the user to delete, or put deletion behind some `betterdiscordctl doctor` & possibly `betterdiscordctl clean`. (`doctor` might fall safely under `status` for us.)

The idea of a `doctor` subcommand is that it tells a human, interactively, what's wrong, possibly wrong, or could use cleaning up. We could then offer to solve parts of that report for the user under `clean`, or upon flags to `doctor`.

Reply:
This safely falls under our existing legacy cleaning functionality.